### PR TITLE
Fix: Solve the complex and heavy-weight bs-platform issue

### PR DESCRIPTION
**Pull request by LeoLeBras**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "files.exclude": {
-    "**/*.bs.js": false
+    "**/*.bs.js": true
```
Comment (LeoLeBras) [inline .vscode/settings.json:3]: Revert this change

File: package.json [hunk 2]
```diff
@@ -36,18 +36,19 @@
...
-    "bs-platform": "^7.2.2",
-    "gentype": "^3.15.0",
-    "jest": "^25.1.0",
-    "ts-jest": "^25.2.1",
-    "typescript": "^3.8.3"
+    "@glennsl/bs-jest": "0.4.9",
+    "@types/jest": "25.1.4",
+    "@types/node": "13.9.1",
+    "babel-jest": "25.1.0",
+    "bs-platform": "git://github.com/rescript-lang/rescript-compiler.git",
```
Comment (LeoLeBras) [inline package.json:43]: Wait a stable release

**Review by LeoLeBras — COMMENTED** [review]
Request some changes

File: package.json (hunk 2 above)
Comment (LeoLeBras) [reply package.json]: Done with https://github.com/winoteam/re-big/pull/2/commits/69c64129dd0919b1a488e712403cd1318f5225ab

File: .vscode/settings.json (hunk 1 above)
Comment (LeoLeBras) [reply .vscode/settings.json]: Done with https://github.com/winoteam/re-big/pull/2/commits/1eb12a737cc31c5842c2158506ca34e83269aeb3

**Review by LeoLeBras — COMMENTED** [review]
It seems ok to me ✅

**Review by gensjaak — APPROVED** [review]
